pkg/flowkit/arguments.go

```diff
@@ -114,3 +120,58 @@ func ParseArguments(args []string, argsJSON string) (scriptArgs []cadence.Value,
 
 	return
 }
+
+func ParseArgumentsWithoutType(code []byte, args []string) (scriptArgs []cadence.Value, err error) {
+
```

Comment:
Nice! There's also `Checker.EntryPointParameters()` in Cadence which might help to reduce the code here a bit